packages/pb/pb.pacscript

```diff
@@ -0,0 +1,17 @@
+name="pb"
+pkgver="0.0.1"
+pkgdesc="0x0 pastebin utility"
+maintainer="vigress8 <vig@disroot.org>"
+url="https://git.envs.net/envs/pb.git"
+pkgver() {
+  git ls-remote "${url}" master | cut -f1 | cut -c1-8
+}
+
+prepare() {
+  git clone "${url}"
```

Comment:
This isn't needed.

Reply:
@Elsie19 when I remove `prepare()` I get this:

```
/tmp/pb.pacscript: line 11: cd: pb: No such file or directory
make: *** No rule to make target 'install'.  Stop.
	[!] ERROR: Could not package pb properly
```

Reply:
remove it and apply the other suggestion I made.

Reply:
@Elsie19 exact same `make` error:
`No rule to make target 'install'.  Stop.`

Reply:
ok, I will look later.